test/HarpoonBattleTest.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * File:   HarpoonBattleTest.cpp
+ * Author: Kjell Hedstrom
+ *
+ * Created on November 30, 2015
+ */
+
+
+#include "HarpoonBattleTest.h"
+#include "Kraken.h"
+#include "KrakenBattle.h"
+#include "HarpoonBattle.h"
+#include "KrakenIntegrationHelper.h"
+#include <Death.h>
+#include <memory>
+#include <tuple>
+#include <utility>
+
+
+TEST_F(HarpoonBattleTest, EnumToString) {
+   auto data = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::Data);
+   auto done = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::Done);
+   auto error = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::Error);
+   auto end = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::End);
+
+   EXPECT_TRUE(data == "<DATA>");
+   EXPECT_TRUE(done == "<DONE>");
+   EXPECT_TRUE(error == "<ERROR>");
+   EXPECT_TRUE(end == "<END>");
+
+}
+
+TEST_F(HarpoonBattleTest, EnumToString_Converted) {
+   auto data = HarpoonBattle::EnumToString(static_cast<HarpoonBattle::ReceivedType>(KrakenBattle::SendType::Data));
+   auto done = HarpoonBattle::EnumToString(static_cast<HarpoonBattle::ReceivedType>(KrakenBattle::SendType::Done));
+   auto error = HarpoonBattle::EnumToString(static_cast<HarpoonBattle::ReceivedType>(KrakenBattle::SendType::Error));
+   auto end = HarpoonBattle::EnumToString(static_cast<HarpoonBattle::ReceivedType>(KrakenBattle::SendType::End));
+
+   EXPECT_TRUE(data == "<DATA>");
+   EXPECT_TRUE(done == "<DONE>");
+   EXPECT_TRUE(error == "<ERROR>");
+   EXPECT_TRUE(end == "<END>");
+}
+
+
+// Disabled since it will trigger a fatal shutdown of the logger
```

Comment:
It doesn't look like this test is disabled. Is the comment wrong, or do you want to disable the test?